.github/actions/load-workflow-variables/action.yml

```diff
@@ -33,20 +33,30 @@ inputs:
 runs:
   using: 'composite'
   steps:
+    - name: 'Sanitize Input Filepath'
```

Comment:
It'd be helpful to document this step inline. There's good documentation in the PR, but we should assume that all that history is lost post-merge :)